content/departments/engineering/teams/repo-management/upgrade-p4-fusion.md

```diff
@@ -38,7 +38,7 @@ You'll need to configure docker so that the images build for amd64, not arm64.
 
 ```sh
 docker buildx create --use
-docker buildx build -t p4-fusion --platform linux/amd64 --target=p4-fusion .
+docker buildx build --load -t p4-fusion --platform linux/amd64 --target=p4-fusion .
```

Comment:
Interesting. Is this flag something that was always needed but didn't have it in the docs? Cannot recall needing this flag the last time I upgraded p4-fusion.

Reply:
Oh I think I recall now. I had some difficulties with building this on OSX and booted up my Linux work machine to do it instead which might explain why I didn't need the flag there. Do you want to clarify that this is for OSX?

Reply:
Haha! Never mind. This **is** a section for OSX.

Reply:
😀😀😀